### PR TITLE
Nonetype error

### PR DESCRIPTION
**Pull request by Smyja**
This fixes the `nonetype` error it throws when the submitted text is problematic.

### CODE REVIEW

File: sense2vec/sense2vec.py [hunk 1]
```diff
@@ -204,6 +204,8 @@ def most_similar(
         """
         if isinstance(keys, (str, int)):
             keys = [keys]
+        if keys == None:
```
Comment (svlandeg) [inline sense2vec/sense2vec.py:207]:
```suggestion
        if keys is None:
```

**Review by svlandeg — COMMENTED** [review]
Thanks for the PR!

Can you add a unit test that breaks before the PR, and is fixed with it?

The typing of `most_similar` doesn't actually allow `None` as a valid value for `keys`, so I wonder when this comes up. If it happens internally, that might be a bug (and we should raise an error instead of silently continuing). If it's not considered a bug, perhaps we need to expand the typing.

**Comment by svlandeg** [comment]
Hi @Smyja, have you had time to look into this question I wrote earlier?

> Can you add a unit test that breaks before the PR, and is fixed with it?
> 
> The typing of `most_similar` doesn't actually allow `None` as a valid value for `keys`, so I wonder when this comes up. If it happens internally, that might be a bug (and we should raise an error instead of silently continuing). If it's not considered a bug, perhaps we need to expand the typing.

**Comment by Smyja** [comment]
> Hi @Smyja, have you had time to look into this question I wrote earlier?
> 
> > Can you add a unit test that breaks before the PR, and is fixed with it?
> > The typing of `most_similar` doesn't actually allow `None` as a valid value for `keys`, so I wonder when this comes up. If it happens internally, that might be a bug (and we should raise an error instead of silently continuing). If it's not considered a bug, perhaps we need to expand the typing.

For badly formatted texts, you'd get a nonetype error. Can't get the tests folder to run without failing, I'll keep trying it.

**Comment by svlandeg** [comment]
> For badly formatted texts, you'd get a nonetype error. Can't get the tests folder to run without failing, I'll keep trying it.

Can you please give an actual code snippet as example?

**Comment by rmitsch** [comment]
Closing this due to lack of a response. @Smyja If you are still working on this, feel free to open a new PR - we'd welcome your contribution! :slightly_smiling_face:

**Pull request closed without merging**